app/controllers/idv/hybrid_mobile/socure/document_capture_controller.rb

```diff
@@ -120,6 +130,31 @@ def analytics_arguments
             selfie_check_required: false,
           }
         end
+
+        def capture_app_analytics(timer)
```

Comment:
- ideally, we'd like to preserver the document_request body and document_response body values as transmitted. it seems like you could leverage document_request.body here.
- in this case it may be a lot simpler to log the analytics in this method
- if these methods are identical in standard and hybrid controllers, perhaps it should move to a concern